Kickstarter-iOS/DataSources/DiscoveryFiltersDataSource.swift

```diff
@@ -5,6 +5,7 @@ internal final class DiscoveryFiltersDataSource: ValueCellDataSource {
   internal enum Section: Int {
     case collectionsHeader
     case collections
+    case categoriesLoader
     case favoritesHeader
```

Comment:
just curious... would we want the loader to appear below the favorites? won't the favorites appear immediately?

or is it not until categories load from the api?